C/coordgenlibs/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "coordgenlibs"
+version = v"1.3.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/schrodinger/coordgenlibs.git", "82b4fe9d13eaa9cd1eabc1acde243c474dff8b40"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/coordgenlibs
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DCOORDGEN_BUILD_TESTS=OFF -DCOORDGEN_BUILD_EXAMPLE=OFF
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```

Comment:
The audit shows the following warning:
```
┌ Warning: /agent/_work/1/s/C/coordgenlibs/build/i686-linux-gnu/HYvPpuwG/destdir/lib/libcoordgen.so.1.3.2 contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder /depot/packages/BinaryBuilder/4fMlX/src/auditor/compiler_abi.jl:189
```
Need to modify
```suggestion
platforms = expand_cxxstring_abis(supported_platforms())
```